pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "Vacancies analysis"
+version = "0.1.0"
+description = "Vacancies analysis with machine learning."
+authors = ["mnv", "aleksandrmurzin", "joein"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.10"
+# парсинг
+Scrapy = "^2.7.0"
+scrapy-user-agents = "^0.1.1"
+# работа с MongoDB
+pymongo = {version = "^4.3.2", extras = ["srv"]}
+
+[tool.poetry.dev-dependencies]
```

Comment:
https://python-poetry.org/docs/master/managing-dependencies/#dependency-groups

Reply:
Added dependency groups splitting the reqs